README.md

```diff
@@ -1,105 +1,80 @@
+**NOTE THE VALUE OF** evapotranspiration_params.zero_plane_displacement_height COMES FROM LAND COVER DATA.  
+ Taken from:    https://websites.pmc.ucsc.edu/~jnoble/wind/extrap/  
+
+| Roughness class  | Roughness length(m)  | Landscape Type                                                                             |  
+| ---------- | ----------- | ------------------------------------------------------------------------------------------- |  
+| 0         | 0.0002     | Smooth water surface                                                                       |  
+| 0.2       | 0.0005     | Inlet water                                                                                |  
+| 0.5       | 0.0024     | Completely open terrain, smooth surface, e.g. concrete runways in airports, mowed grass, etc.|  
+| 1         | 0.03       | Open agricultural area without fences and hedgerows and very scattered buildings. Only softly rounded hills |  
+| 1.5       | 0.055      | Agricultural land with some houses and 8 metre tall sheltering hedgerows with a distance of approximately 1250 metres |  
+| 2         | 0.1        | Agricultural land with some houses and 8 metre tall sheltering hedgerows with a distance of approximately 500 metres |  
+| 2.5       | 0.2        | Agricultural land with many houses, shrubs and plants, or 8 metre tall sheltering hedgerows with a distance of approximately 250 metres |  
+| 3         | 0.4        | Villages, small towns, agricultural land with many or tall sheltering hedgerows, forests and very rough and uneven terrain |  
+| 3.5       | 0.8        | Larger cities with tall buildings|  
+| 4         | 1.6        | Very large cities with tall buildings and skyscrapers|  
+
+Roughness definitions according to the European Wind Atlas.  
+According to the UN FAO Penman-Monteith example here: http://www.fao.org/3/X0490E/x0490e06.htm#aerodynamic%20resistance%20(ra)  
+The zero plane roughness length,"d" can be approximated as 2/3 of the vegetation height (H): d=2/3*H.  
+The momentum roughness height "zom" can be estimated as 0.123*H.  
+The heat transfer roughness height "zoh" can be approximated as 0.1 * zom.  
+
+# A note on code adaptation for BMI
+This code was minimally changed from the author's original version. These minor changes were made by NGen NWM formulation team:
+* Much of this C code was moved to `*.h` files, with the intention of being more easily integrated into the NGen Framework. It turned out that this step was not stricktly neccessary, and that this standalone module could use the standard `*.c` files. See known issues below for a discussion on turning these back to `*.c` files.
```

Comment:
typos: strictly necessary

Reply:
I kind of liked that spelling of 'stricktly'